legal-api/src/legal_api/resources/business/business_directors.py

```diff
@@ -55,8 +55,6 @@ def get(identifier, director_id=None):
                 del director_json['mailingAddress']
             res.append(director_json)
 
-        if not res:
```

Comment:
Are there unit tests that need to be updated (or written)?

Reply:
you know what, I asked myself the same question but didn't see any specific coverage for this and pytest ran clean...but on second look I do see a test case for this. Updating now... (I ran pytest without the make command and it didn't run the suite that would have failed)